lib/livebook_cli/deploy.ex

```diff
@@ -0,0 +1,221 @@
+defmodule LivebookCLI.Deploy do
+  import LivebookCLI.Utils
+  alias Livebook.Teams
+
+  @behaviour LivebookCLI.Task
+
+  @deploy_key_prefix Teams.Requests.deploy_key_prefix()
+  @teams_key_prefix Teams.Org.teams_key_prefix()
```

Comment:
It would be nice if we could have these in a single place. What about having a `Teams.Prefixes` module or `Teams.Constants` module which only has this stuff? This will also help keep compile cycles small, as those files won't have any logic in them.